package.json

```diff
@@ -10,7 +10,7 @@
     "build:dist": "NODE_ENV=production webpack --config webpack.config.production.js",
     "lint": "eslint --ext .js,.jsx src",
     "deploy": "npm run build:dist && surge -p .",
-    "start": "node server.js"
+    "start": "NODE_ENV=development node server.js"
```

Comment:
Awesome catch @ryan-roemer , fixed here a32123c36ee9f47d6e7fbd61c3f95820d6ee43b8 

Does it make sense to build a Spectacle builder archetype for core too? Or just for the boilerplate?